src/std/sugar-test.ss

```diff
@@ -1,6 +1,7 @@
 (export sugar-test)
 
 (import :std/test
+        :std/srfi/115
```

Comment:
lets use pregexp for this, i find it much more ergonomic and familiar.

Reply:
Yes.